packages/flutter/test/cupertino/refresh_test.dart

```diff
@@ -1327,48 +1327,40 @@ void main() {
         );
     }, variant: const TargetPlatformVariant(<TargetPlatform>{ TargetPlatform.iOS,  TargetPlatform.macOS }));
 
-    testWidgets('buildSimpleRefreshIndicator dark mode', (WidgetTester tester) async {
-      const CupertinoDynamicColor color = CupertinoColors.inactiveGray;
-
+    testWidgets('buildAppleRefreshIndicator progress', (WidgetTester tester) async {
```

Comment:
Why has this test changed?

Reply:
The old test used to check for the presence of an icon (the down-arrow) and make sure that the colour of the icon was correct. The change introduced in this PR (which matches iOS implementation) doesn't have an icon any more, so now the test makes sure that the in-progress calculations are correct.